src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,69 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { useState } from 'react';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const CENTURIES_VALUE: number[] = [16, 17, 18, 19, 20];
```

Comment:
I would keep it outside of the component, typically in some kind of config file.

BTW you don't need that `number[]` here. TS would figure it out by itself ;)